test/test_boundary_conditions_integration.jl

```diff
@@ -227,7 +227,8 @@ function test_open_boundary_condition_mass_conservation(arch, FT, boundary_condi
     Δt = 0.1 * minimum_zspacing(grid) / maximum(abs, u)
     simulation = Simulation(model; stop_time=1, Δt, verbose=false)
 
-    ∫∇u = Field(Integral(∂x(u) + ∂y(v) + ∂z(w)))
+    ∇u = Field(∂x(u) + ∂y(v) + ∂z(w))
+    ∫∇u = Field(Integral(∇u))
```

Comment:
You should be able to define `Integral` of the divergence directly without using an intermediate `Field`.

Reply:
That's what I had before. It compiled on my local machines, but it wasn't compiling for GPU on the CI so I changed it in the last commit. It seems to work now.